package-lock.json

```diff
@@ -10291,8 +11532,18 @@
         "json5": "lib/cli.js"
       }
     },
+    "node_modules/tsconfig-paths/node_modules/strip-bom": {
+      "version": "3.0.0",
+      "resolved": "https://registry.npmjs.org/strip-bom/-/strip-bom-3.0.0.tgz",
+      "integrity": "sha512-vavAMRXOgBVNF6nyEEmL3DBK19iRpDcoIwW+swQ+CbGiu7lju6t+JklA1MHweoWtadgt4ISVUsXLyDq34ddcwA==",
+      "dev": true,
+      "engines": {
+        "node": ">=4"
+      }
+    },
     "node_modules/tsdx": {
       "version": "0.14.1",
+      "resolved": "https://registry.npmjs.org/tsdx/-/tsdx-0.14.1.tgz",
```

Comment:
looks like `tsdx` Installation changes are still in package-lock.json
please remove all changes from this file in the PR

Reply:
Yes, that's because `tsdx` is still a dev dependency (in devDependencies). Should I remove it from the project entirely?
And the tests are run by `tsdx`.

Reply:
no, leave it in for now, we have another task to remove it

Reply:
Okay. So does that mean this PR is completed?

Reply:
Okay